src/components/Avatar/__tests__/avatar-snapshot-tests.jest.tsx

```diff
@@ -3,7 +3,8 @@ import renderer from "react-test-renderer";
 import Avatar from "../Avatar";
 import { WhatsNew } from "../../Icon/Icons";
 
-jest.mock("../../Icon/Icon", () => ({ icon }) => <div data-testid="icon-mock">{icon}</div>);
+// TODO: Add type for icon
```

Comment:
@orrgottlieb Could you please help me here with the type for the icon

Reply:
@SumeetHaryani you can define - 
`export type IconType =  string | React.FunctionComponent | null;` in types, and use this

Reply:
thanks @orrgottlieb